packages/params/src/interface/phase0.ts

```diff
@@ -16,7 +15,7 @@ export interface IPhase0Preset {
   SAFE_SLOTS_TO_UPDATE_JUSTIFIED: number;
 
   // Gwei Values
-  MIN_DEPOSIT_AMOUNT: bigint;
+  MIN_DEPOSIT_AMOUNT: number;
```

Comment:
This isn't used anywhere?! I guess its hard to argue against changing it to a number

Reply:
Yeah, not used anywhere 🤷